exchanges/alert/alert.go

```diff
@@ -61,8 +61,8 @@ func (n *Notice) Wait(kick <-chan struct{}) <-chan bool {
 	return reply
 }
 
-// hold waits on either channel in the event that the routine has finished or an
-// alert from a depth update has occurred.
+// hold waits on either channel in the event that the routine has
+// finished/canceled or an alert from a depth update has occurred.
```

Comment:
This is incredibly minor, but we have 248 usages of `cancelled` in our codebase and only 53 of `canceled` additionaly, the `canceled` usage is interpreting foreign APIs. Can we use `cancelled`?